axelrod/strategies/zero_determinant.py

```diff
@@ -225,3 +227,77 @@ class ZDSet2(LRPlayer):
 
     def __init__(self, phi: float = 1 / 4, s: float = 0.0, l: float = 2) -> None:
         super().__init__(phi, s, l)
+
+
+class AdaptiveZeroDet(LRPlayer):
+    """A strategy that uses a zero determinant structure that updates
+    its parameters after each round of play.
```

Comment:
It would be helpful to have some more information here, perhaps an explanation of the update procedure? It's ok to go quite in depth and include some mathematics which we can then use to check that the `_adjust_parameters` method is working as expected.